netbox/dcim/migrations/0182_alter_rack_starting_unit.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.5 on 2023-12-05 19:13
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('dcim', '0181_rename_device_role_device_role'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='rack',
+            name='starting_unit',
+            field=models.PositiveSmallIntegerField(default=1, validators=[django.core.validators.MinValueValidator(1)]),
```

Comment:
Because the implementation of a field validator doesn't actually effect a concrete change in the database, we can move this into the migration file where the field is defined (`0174_rack_starting_unit`). This allows us to avoid introducing a no-op migration file.

(The only check constraint on the `starting_unit` field is for `>= 0`, which is installed by PositiveSmallIntegerField.)

Reply:
Done.